lib/puppet/provider/archive/ruby.rb

```diff
@@ -80,6 +80,7 @@ def exists?
   def create
     transfer_download(archive_filepath) unless checksum?
     extract
+  ensure
```

Comment:
whats enaure doing here, is that a ruby internal as well? if its a simple method call, shouldnt it be indented more?

Reply:
Yes, `ensure` is a ruby keyword. It marks a block of code that must run before the method exits, even if an exception is raised.  It gets indented the same way as a `rescue` block.